### ISSUE TITLE
HTMLExecutingFunctionsSniff: Add more unsafe functions to $HTMLExecutingFunctions

### ISSUE THREAD

**Issue by rebeccahum** [Automattic/VIP-Coding-Standards#686]
## Describe the solution you'd like

We should add to the `$HTMLExecutingFunctions` property the following functions:

- insertAdjacentHTML
- outerHTML

## What code should be reported as a violation?

```
(function(){
	divElem.outerHTML = "<div>Hello " + userControlledVal + "</div>"; // Warning.
	divElem.insertAdjacentHTML("","<b>"+ userControlledVal + "</b>"); // Warning.
})();
```

## Additional context
http://blog.blueclosure.com/2017/09/

**Comment by GaryJones** [comment]
As the JavaScript sniffs have been deprecated for VIPCS 3.1.0, I'm closing this out.
